Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.14 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:3.12.4
+ENV HELM_VERSION="v3.3.1"
+RUN wget -q https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm && chmod +x /usr/local/bin/helm
```

Comment:
Can we do this in the builder stage itself? We don't need the helm binary in the final image